src/proto/common.proto

```diff
@@ -0,0 +1,82 @@
+// API for the client to interact with the coordinator.
+
+syntax = "proto3";
+
+package common;
+
+import "google/protobuf/timestamp.proto";
+
+// Identifies which cohort is responsible for executing the operation.
+message Namespace {
+  string identifier = 1;
+}
+
+// Using a message to allow for extending to support multiple keys (e.g.
+// Get(StartsWith("foo"))).
+message Get {
+  // Scoped within the current namespace.
+  string key = 1;
+}
+
+message ConstantValue {
+  oneof type {
+    int64 int64_value = 1;
+    double double_value = 2;
+    string string_value = 3;
+    // Can add more types here (e.g. bytes, lists, etc.)
+  }
+}
+
+message Value {
+  // Values can either be from a get operation or a constant value.
+  // Add support for Put("b", Get("a")) operations later.
```

Comment:
Should we create an GitHub issue to track these? It seems to be something we can do if we have extra time.

Reply:
Created https://github.com/benjmarks22/blockchain-2pc/issues/3